mysql-test/suite/rocksdb/t/select_for_update.test

```diff
@@ -1,21 +1,29 @@
---source include/have_rocksdb_as_default.inc
+--source include/have_rocksdb.inc
 
 #
 # SELECT .. FOR UPDATE
 #
 
---disable_warnings
-DROP TABLE IF EXISTS t1;
---enable_warnings
-
 --enable_connect_log
 
 --source include/count_sessions.inc
 
+# MyRocks does not support gap locks in REPEATABLE-READ mode, this test
+# does not require RR ISO to complete, so lets temporarily alter the ISO
+# to RC
+set @orig_tx_iso=@@session.tx_isolation;
+set session tx_isolation='READ-COMMITTED';
+
 CREATE TABLE t1 (a INT, b CHAR(8), pk INT AUTO_INCREMENT PRIMARY KEY) ENGINE=rocksdb;
 INSERT INTO t1 (a,b) VALUES (1,'a'),(2,'b'),(3,'a');
 
 --connect (con1,localhost,root,,)
+# MyRocks does not support gap locks in REPEATABLE-READ mode, this test
+# does not require RR ISO to complete, so lets temporarily alter the ISO
+# to RC
+set @orig_tx_iso=@@session.tx_isolation;
```

Comment:
Saving the session variable is twice as much not required, when you just opened a session :)